src/cloud/utils/reporting.ts

```diff
@@ -184,3 +188,26 @@ export const useLoadTimeReporting = (measurement: string) => {
     })
   }, [measurement, loadStartTime])
 }
+
+export const updateCampaignInfo = (query: string) => {
+  if (CLOUD && isFlagEnabled('rudderstackReporting')) {
+    const allowedUtmKeys = ['campaign', 'term', 'source', 'medium', 'content']
+    const queryParams = new URLSearchParams(query)
+    const info = {}
+    queryParams.forEach(function(value, key) {
+      if (key.startsWith('utm_')) {
+        let param = key.substr(4)
+        if (allowedUtmKeys.includes(param)) {
+          // see rudder-sdk-js (#2963)
```

Comment:
is it possible to provide a direct link to the issue?

Reply:
Added link to source code (sorry using #N was misleading).